src/pages/ResetPassword/ResetPassword.css

```diff
@@ -0,0 +1,31 @@
+.reset-password-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.reset-info-text {
+  margin: 0;
+}
+
+.reset-passwords-container {
+  align-self: center;
+  gap: 2vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.reset-input-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 2vh;
+}
+
+.reset-submit-button {
+  align-self: center;
+  background-color: #21307a;
```

Comment:
although the jury is still out on `global.css` vs. chakra theme colors, for now since we have the file we should define common colors like cds blue 1 (#21307a) by importing the `global.css` stylesheet from `src/common` and then using a css var so that in case #21307a changes, we don't have to go through the work to redefine cds blue 1 everywhere it's being used

to import a stylesheet, at the top of the page:
```CSS
@import 'path/to/file';
```

example of using a css var:
```CSS
.reset-submit-button {
    ...
    background-color: var(--cds-blue-1);
    ...
}
```